src/app.js

```diff
@@ -37,18 +38,18 @@ async function getCountry(countries) {
 
         const printInputCountries = (countries) => {
```

Comment:
Bepaalde functies zijn ook om te schrijven naar de filter method (of zelfs de map method als je dan de true uitkomst van een object d.m.v. een statement pusht in een array) maar ik vind de for loops zelf mooier omdat dat laat zien wat deze methods doen onder de motorkap!